packages/block-library/src/cover/edit.native.js

```diff
@@ -514,7 +514,10 @@ const Cover = ( {
 				pointerEvents="box-none"
 				style={ [ styles.content, { minHeight: CONTAINER_HEIGHT } ] }
 			>
-				<InnerBlocks template={ INNER_BLOCKS_TEMPLATE } />
+				<InnerBlocks
+					template={ INNER_BLOCKS_TEMPLATE }
+					templateInsertUpdatesSelection={ true }
```

Comment:
It should be equivalent with:
```suggestion
					templateInsertUpdatesSelection
```